browser/importer/brave_profile_writer.cc

```diff
@@ -441,3 +447,72 @@ void BraveProfileWriter::UpdateWindows(
   if (active)
     active->window()->Show();
 }
+
+
+// NOTE: the strings used as keys match the values found in Muon:
+// browser-laptop/js/data/searchProviders.js
+std::map<std::string,TemplateURLPrepopulateData::PrepopulatedEngine>
```

Comment:
Just nitpicks, seems like this could be: static const std::map<std::string, const TemplateURLPrepopulateData::PrepopulatedEngine>